internal/service/apprunner/vpc_connector_test.go

```diff
@@ -167,70 +184,62 @@ resource "aws_vpc" "test" {
   }
 }
 
-resource "aws_security_group" "test" {
-  vpc_id = aws_vpc.test.id
-  name   = %[1]q
+resource "aws_subnet" "test" {
+  cidr_block = "10.1.1.0/24"
+  vpc_id     = aws_vpc.test.id
 
   tags = {
     Name = %[1]q
   }
 }
 
-resource "aws_subnet" "test" {
-  cidr_block = "10.1.1.0/24"
-  vpc_id     = aws_vpc.test.id
+resource "aws_security_group" "test" {
+  vpc_id = aws_vpc.test.id
 
   tags = {
     Name = %[1]q
   }
 }
+`, rName)
+}
 
+func testAccVPCConnectorConfig_basic(rName string) string {
+	return acctest.ConfigCompose(testAccVPCConnectorConfig_base(rName), fmt.Sprintf(`
 resource "aws_apprunner_vpc_connector" "test" {
   vpc_connector_name = %[1]q
   subnets            = [aws_subnet.test.id]
   security_groups    = [aws_security_group.test.id]
 }
-`, rName)
+`, rName))
 }
 
 func testAccVPCConnectorConfig_tags1(rName string, tagKey1 string, tagValue1 string) string {
-	return fmt.Sprintf(`
-resource "aws_vpc" "test" {
-  cidr_block = "10.1.0.0/16"
-
-  tags = {
-    Name = %[1]q
-  }
-}
-
-resource "aws_security_group" "test" {
-  vpc_id = aws_vpc.test.id
-  name   = %[1]q
+	return acctest.ConfigCompose(testAccVPCConnectorConfig_base(rName), fmt.Sprintf(`
+resource "aws_apprunner_vpc_connector" "test" {
+  vpc_connector_name = %[1]q
+  subnets            = [aws_subnet.test.id]
+  security_groups    = [aws_security_group.test.id]
 
   tags = {
-    Name = %[1]q
+    %[2]q = %[3]q
   }
 }
-
-resource "aws_subnet" "test" {
-  cidr_block = "10.1.1.0/24"
-  vpc_id     = aws_vpc.test.id
-
-  tags = {
-    Name = %[1]q
-  }
+`, rName, tagKey1, tagValue1))
 }
 
+func testAccVPCConnectorConfig_tags2(rName string, tagKey1 string, tagValue1 string, tagKey2 string, tagValue2 string) string {
+	return acctest.ConfigCompose(testAccVPCConnectorConfig_base(rName), fmt.Sprintf(`
 resource "aws_apprunner_vpc_connector" "test" {
   vpc_connector_name = %[1]q
   subnets            = [aws_subnet.test.id]
   security_groups    = [aws_security_group.test.id]
 
   tags = {
-    %[2]q = %[3]q
+	%[2]q = %[3]q
+	%[4]q = %[5]q
   }
```

Comment:
styling change for `terrafmt`

```suggestion
  tags = {
    %[2]q = %[3]q
    %[4]q = %[5]q
  }